src/ui/popup/regex.tsx

```diff
@@ -62,7 +62,15 @@ export default function Regex(props: {
 	tab: Resource<ManagerTab>;
 }) {
 	return (
-		<div class={styles.regexContainer}>
+		<div
+			class={styles.regexContainer}
+			onKeyDown={(event) => {
+				console.log(event);
```

Comment:
```suggestion
```

Reply:
removed this console.log here -> https://github.com/web-scrobbler/web-scrobbler/pull/3785/commits/d5d993d8a34833f897bbcf2a61196ee5cd363ab1